### PR TITLE
feat(design-system): Skeleton CSS module conversion

### PR DESCRIPTION
**Pull request by Benrajalu**
**What is the problem this PR is trying to solve?**

Skeletons are not CSS module nor using Tokens. They don't align with current components well.

**What is the chosen solution to this problem?**

Move them to CSS modules and make them more modular.

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/design-system/src/components/Skeleton/variations/SkeletonButton.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+};
+
+const SkeletonButton = forwardRef(
+	({ size = 'M', ...props }: SkeletonButtonProps, ref: Ref<HTMLSpanElement>) => {
+		return (
+			<SkeletonPrimitive
+				className={classNames(styles.buttonSkeleton, {
+					[PrimitiveStyles['size-XL']]: size === 'M',
+					[PrimitiveStyles['size-L']]: size === 'S',
+					[styles.small]: size === 'S',
```
[the remark this answers is not in the document]
Comment (Benrajalu) [reply packages/design-system/src/components/Skeleton/variations/SkeletonButton.tsx:19]: Wouldn't have this issue with different scales for sizing and spacing...

File: packages/design-system/src/components/Skeleton/Skeleton.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import Skeleton, { SkeletonButton, SkeletonParagraph, SkeletonHeading } from '.';
+import { StackHorizontal, StackVertical } from '../Stack';
+
+import * as Stories from './Skeleton.stories';
+
+<Meta
+	title="Components/Skeleton"
+	parameters={{
+		status: { figma: 'wip', storybook: 'wip', react: 'ok', i18n: 'na' },
```
Comment (frassinier) [inline packages/design-system/src/components/Skeleton/Skeleton.stories.mdx:10]: Don't forget to update that status and to add your future Figma link too

File: packages/design-system/src/components/Skeleton/Skeleton.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	<Story story={Stories.SkeletonParagraphStory} name="Paragraphs" />
+</Canvas>
+
+<ArgsTable story="Paragraphs" />
+
+#### Composition example
+
+Combine all of them to mock content!
+
+<Canvas>
```
Comment (frassinier) [inline packages/design-system/src/components/Skeleton/Skeleton.stories.mdx:64]: Why not using a real TSX for that?

File: packages/design-system/src/index.ts [hunk 4]
```diff
@@ -100,6 +105,10 @@ export {
 	StatusWarning,
 	StatusSuccessful,
 	StatusFailed,
+	SkeletonParagraph,
+	SkeletonHeading,
+	SkeletonButton,
+	SkeletonButtonIcon,
```
Comment (frassinier) [inline packages/design-system/src/index.ts:111]: Do you really hate me? I tried to keep them alpha sorted! 🤣

**Review by frassinier — APPROVED** [review]

File: packages/design-system/src/components/Skeleton/Skeleton.stories.mdx (hunk 3 above)
Comment (Benrajalu) [reply packages/design-system/src/components/Skeleton/Skeleton.stories.mdx:64]: Laziness

File: packages/design-system/src/index.ts (hunk 4 above)
Comment (Benrajalu) [reply packages/design-system/src/index.ts:111]: Haha ok I'll try to as well!

File: packages/design-system/.storybook/preview.js [hunk 5]
```diff
@@ -12,8 +12,6 @@ import { DocsContainer } from '@storybook/addon-docs';
 import { UPDATE_GLOBALS, SET_STORIES } from '@storybook/core-events';
 import { TableOfContents, BackToTop } from 'storybook-docs-toc';
 
-import '@talend/design-tokens/lib/index.scss';
-
```
Comment (Benrajalu) [inline packages/design-system/.storybook/preview.js:16]: Removed to put it in the ThemeProvider

File: packages/design-system/src/components/ThemeProvider/ThemeProvider.tsx [hunk 6]
```diff
@@ -4,6 +4,8 @@ import {
 	ThemeProvider as StyledThemeProvider,
 	ThemeProviderProps,
 } from 'styled-components';
+// eslint-disable-next-line @talend/import-depth
+import '@talend/design-tokens/dist/TalendDesignTokens.css';
```
Comment (Benrajalu) [inline packages/design-system/src/components/ThemeProvider/ThemeProvider.tsx:8]: Has to be the CSS in order to avoid recompiling and hashing keyframes.

**Pull request merged**